shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -841,6 +857,7 @@ public boolean performAction(int virtualViewId, int accessibilityAction, @Nullab
                     AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUS_CLEARED
                 );
                 accessibilityFocusedSemanticsNode = null;
+                embeddedFocusedNodeId = null;
```

Comment:
Is this line actually required? Wouldn't the event to null this out always be handled by the if you added above?